Lesson3/Sulima Homework 2/src/functions.cpp

```diff
@@ -0,0 +1,91 @@
+#include "functions.h"
+#include <stack>
+#include <queue>
+
+char reverseRightToLeftBracketsFunction(const char m_rightBrackets){
+    switch (m_rightBrackets)
+    {
+    case '}':
+    {
+        return '{';
+    }
+    case ')':
+    {
+        return '(';
+    }
+    case ']':
+    {
+        return '[';
+    }
+    default:
+        return NULL;
```

Comment:
_exception?_

Reply:
yes, if we pass a no bracket to the function, we get NULL.